csharp/extractor/Semmle.Extraction.CSharp.Standalone/Runtime.cs

```diff
@@ -66,7 +66,7 @@ private static IEnumerable<string> DesktopRuntimes
         /// <summary>
         /// Gets the .NET runtime location to use for extraction
         /// </summary>
-        public static string GetRuntime(bool useSelfContained) => useSelfContained ? ExecutingRuntime : Runtimes.First();
+        public static string GetRuntime(bool useSelfContained) => /*useSelfContained ? ExecutingRuntime :*/ Runtimes.First();
```

Comment:
Why did you have to disable this?

Reply:
(It is only disabled temporarily.) I disabled it to see if the standalone test would be fixed by removing the DB constraint. I think it only reproduced on Jenkins and not locally with the embedded runtime.

Reply:
Ahh, okay, I see.